plugins/package-managers/python/src/main/kotlin/Poetry.kt

```diff
@@ -51,6 +51,13 @@ class Poetry(
     analyzerConfig: AnalyzerConfiguration,
     repoConfig: RepositoryConfiguration
 ) : PackageManager(name, analysisRoot, analyzerConfig, repoConfig), CommandLineTool {
+    companion object {
+        /**
+         * The name of the build system requirements and information file used by modern Python packages.
+         */
+        internal const val PYPROJECT_FILENAME = "pyproject.toml"
```

Comment:
FYI, Pip actually also uses `pyproject.toml` (implicitly), see https://pip.pypa.io/en/stable/reference/build-system/pyproject-toml/ and the commit message of ed0838125c562d171f48b1d2df0077e09e866cc6. So it's not wrong per se to keep the variable at `Pip`-scope, but from our current code base I see how it makes sense to move the constant. Overall, I'm undecided about the move.

Reply:
I preferred to move it because in our code base it is really only used by `Poetry` currently.

Reply:
Mind referring to ed0838125c562d171f48b1d2df0077e09e866cc6 then in order to document that you're aware of the original motivation to put this into `Pip`, but want to change that?